### PR TITLE
Finally k8s is working

### PR DESCRIPTION
**Pull request by campidelli**

### CODE REVIEW

File: helpdesk-bot/docker/k8s_deployment.yaml.template [hunk 1]
```diff
@@ -67,15 +71,15 @@ spec:
             - name: PROVISIONING_CA_OVERWRITE
               value: 'true'
             - name: PROVISIONING_SERVICE_ACCOUNT_NAME
-              value: helpdesk-bot
+              value: zezao
```
Comment (robson-sanchez) [inline helpdesk-bot/docker/k8s_deployment.yaml.template]: Please keep 'helpdesk-bot' by default.

File: helpdesk-bot/docker/k8s_deployment.yaml.template [hunk 2]
```diff
@@ -67,15 +71,15 @@ spec:
...
             - name: PROVISIONING_SERVICE_ACCOUNT_GENERATE_KEYSTORE
               value: 'true'
             - name: PROVISIONING_SERVICE_ACCOUNT_OVERWRITE
               value: 'true'
             - name: CERTS_DIR
               value: /opt/symphony/helpdesk-bot/certs
             - name: AUTHENTICATION_KEYSTORE_FILE
-              value: /opt/symphony/helpdesk-bot/certs/helpdesk.p12
+              value: /opt/symphony/helpdesk-bot/certs/zezao.p12
```
Comment (robson-sanchez) [inline helpdesk-bot/docker/k8s_deployment.yaml.template]: Please keep 'helpdesk.p12' by default.

File: helpdesk-service/docker/k8s_deployment.yaml.template [hunk 3]
```diff
@@ -40,5 +42,3 @@ spec:
               value: 'true'
             - name: APP_AUTHENTICATION_FILTER_ENABLED
               value: 'false'
-            - name: MONGO_IGNORESSL
```
Comment (robson-sanchez) [inline helpdesk-service/docker/k8s_deployment.yaml.template:43]: You should include two new environment variables to support app authentication: APP_AUTHENTICATION_API_KEYSTORE_FILE and APP_AUTHENTICATION_API_KEYSTORE_PASSWORD

File: helpdesk-service/helpdesk-service-server/src/main/resources/application.yaml [hunk 4]
```diff
@@ -13,9 +13,10 @@ mongo:
   waitTimeout: 10000
   readPreference: nearest
   writeConcern: majority
+  ignoreSSL: false
 
 app-authentication:
-  enabled: true
+  enabled: false
```
Comment (robson-sanchez) [inline helpdesk-service/helpdesk-service-server/src/main/resources/application.yaml]: This flag should be set to true.

**Review by robson-sanchez — CHANGES_REQUESTED** [review]

File: helpdesk-bot/docker/k8s_deployment.yaml.template (hunk 1 above)
Comment (campidelli) [reply helpdesk-bot/docker/k8s_deployment.yaml.template]: Done.

File: helpdesk-bot/docker/k8s_deployment.yaml.template (hunk 2 above)
Comment (campidelli) [reply helpdesk-bot/docker/k8s_deployment.yaml.template]: Done.

File: helpdesk-service/docker/k8s_deployment.yaml.template (hunk 3 above)
Comment (campidelli) [reply helpdesk-service/docker/k8s_deployment.yaml.template:43]: Done.

File: helpdesk-service/helpdesk-service-server/src/main/resources/application.yaml (hunk 4 above)
Comment (campidelli) [reply helpdesk-service/helpdesk-service-server/src/main/resources/application.yaml]: Done.

**Review by robson-sanchez — APPROVED** [review]

**Pull request merged**